www/index.html

```diff
@@ -330,6 +484,7 @@ <h3>Expert settings</h3>
                         <div class="card card-danger" id="contentInjectionModeDiv">
                             <div class="card-header">Content injection mode</div>
                             <div class="card-body">
+                                <p>See <a href="#modes" id="modesLink">About (Expert Settings)</a> for an explanation of the difference between these modes:</p>
```

Comment:
Strangely, it does for me. Maybe it's browser-dependent as to whether the onclick event cancels the navigation to #modes . Possibly some browsers won't navigate to it until its containing section is unhidden. In which case I can add a scrollto programmatically.

Reply:
OK, I've discovered this problem has nothing to do with the browser. It's the "Animate transition between app pages" that blocks the scroll (or overrides it). Since I have that turned off (it makes me feel slightly dizzy!), I hadn't tested it like that. I can deal with it by having a timeout on the scroll, or else somehow pass the element we want to scroll to to that function.

Reply:
It's a wider issue with any scrollIntoView action (or # anchor link) conflicting with the transition. It's a regression we didn't notice before when the transition animation was introduced. It also affects the links in the "Unable to display active content" warning we show with ZIMs that have a proprietary UI. Probably a separate issue. I have introduced a temporary workaround with a 600ms timeout in this case, but it needs a proper bugfix for the issue more broadly.